protos/sabre_payload.proto

```diff
@@ -0,0 +1,159 @@
+// Copyright 2018-2020 Cargill Incorporated
+//
+// Modifications copyright (C) 2020 Tyson Foods, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+// Modifications copyright (C) 2020 Tyson Foods, Inc.
```

Comment:
Remove this notice - the one at the top of the file is sufficient when combined with git history to determine copyright of specific lines.